### PR TITLE
[SPA] Update npm depencencies for angular and react

### PR DESCRIPTION
**Pull request by javiercn**

### CODE REVIEW

File: src/content/Angular-CSharp/ClientApp/package.json [hunk 1]
```diff
@@ -2,33 +2,33 @@
...
+      "ng": "ng",
 //#if(RequiresHttps)
-    "prestart": "node aspnetcore-https",
-    "start": "run-script-os",
-    "start:windows": "ng serve --port 5002 --ssl --ssl-cert %APPDATA%\\ASP.NET\\https\\%npm_package_name%.pem --ssl-key %APPDATA%\\ASP.NET\\https\\%npm_package_name%.key",
-    "start:default": "ng serve --port 5002 --ssl --ssl-cert $HOME/.aspnet/https/${npm_package_name}.pem --ssl-key $HOME/.aspnet/https/${npm_package_name}.key",
+      "prestart": "node aspnetcore-https",
+      "start": "run-script-os",
+      "start:windows": "ng serve --port 5002 --ssl --ssl-cert %APPDATA%\\ASP.NET\\https\\%npm_package_name%.pem --ssl-key %APPDATA%\\ASP.NET\\https\\%npm_package_name%.key",
+      "start:default": "ng serve --port 5002 --ssl --ssl-cert $HOME/.aspnet/https/${npm_package_name}.pem --ssl-key $HOME/.aspnet/https/${npm_package_name}.key",
```
Comment (javiercn) [inline src/content/Angular-CSharp/ClientApp/package.json:10]:
```suggestion
    "prestart": "node aspnetcore-https",
    "start": "run-script-os",
    "start:windows": "ng serve --port 5002 --ssl --ssl-cert %APPDATA%\\ASP.NET\\https\\%npm_package_name%.pem --ssl-key %APPDATA%\\ASP.NET\\https\\%npm_package_name%.key",
    "start:default": "ng serve --port 5002 --ssl --ssl-cert $HOME/.aspnet/https/${npm_package_name}.pem --ssl-key $HOME/.aspnet/https/${npm_package_name}.key",
```

File: src/content/Angular-CSharp/ClientApp/package.json [hunk 2]
```diff
@@ -2,33 +2,33 @@
...
+      "start": "ng serve --port 5002",
 //#endif
-    "build": "ng build",
-    "build:ssr": "ng run Company.WebApplication1:server:dev",
-    "watch": "ng build --watch --configuration development",
-    "test": "ng test"
+      "build": "ng build",
+      "build:ssr": "ng run Company.WebApplication1:server:dev",
+      "watch": "ng build --watch --configuration development",
+      "test": "ng test"
```
Comment (javiercn) [inline src/content/Angular-CSharp/ClientApp/package.json:17]:
```suggestion
    "build": "ng build",
    "build:ssr": "ng run Company.WebApplication1:server:dev",
    "watch": "ng build --watch --configuration development",
    "test": "ng test"
```

File: eng/Versions.props [hunk 3]
```diff
@@ -9,19 +9,19 @@
     <!-- Packages from dotnet/arcade -->
     <MicrosoftDotNetBuildTasksTemplatingVersion>6.0.0-beta.21373.11</MicrosoftDotNetBuildTasksTemplatingVersion>
     <!-- Packages from dotnet/aspnetcore -->
-    <MicrosoftAspNetCoreApiAuthorizationIdentityServerVersion>6.0.0-preview.6.21355.2</MicrosoftAspNetCoreApiAuthorizationIdentityServerVersion>
-    <MicrosoftAspNetCoreDiagnosticsEntityFrameworkCoreVersion>6.0.0-preview.6.21355.2</MicrosoftAspNetCoreDiagnosticsEntityFrameworkCoreVersion>
-    <MicrosoftAspNetCoreIdentityEntityFrameworkCoreVersion>6.0.0-preview.6.21355.2</MicrosoftAspNetCoreIdentityEntityFrameworkCoreVersion>
-    <MicrosoftAspNetCoreIdentityUIVersion>6.0.0-preview.6.21355.2</MicrosoftAspNetCoreIdentityUIVersion>
-    <MicrosoftAspNetCoreSpaProxyVersion>6.0.0-preview.6.21355.2</MicrosoftAspNetCoreSpaProxyVersion>
-    <MicrosoftAspNetCoreSpaServicesExtensionsVersion>6.0.0-preview.6.21355.2</MicrosoftAspNetCoreSpaServicesExtensionsVersion>
+    <MicrosoftAspNetCoreApiAuthorizationIdentityServerVersion>6.0.0</MicrosoftAspNetCoreApiAuthorizationIdentityServerVersion>
```
Comment (pranavkm) [inline eng/Versions.props:12]: Out of curiosity, how does this work in the template package? Do these versions get stomped over when it's consumed as a submodule in aspnetcore?

**Review by pranavkm — APPROVED** [review]

File: eng/Versions.props (hunk 3 above)
Comment (javiercn) [reply eng/Versions.props:12]: Yeah, these are only for local development

Comment (TanayParikh) [reply eng/Versions.props:12]: Should we consider using the patched version `6.0.1`?

File: src/content/Angular-CSharp/ClientApp/package.json [hunk 4]
```diff
@@ -9,7 +9,7 @@
     "start:windows": "ng serve --port 5002 --ssl --ssl-cert %APPDATA%\\ASP.NET\\https\\%npm_package_name%.pem --ssl-key %APPDATA%\\ASP.NET\\https\\%npm_package_name%.key",
     "start:default": "ng serve --port 5002 --ssl --ssl-cert $HOME/.aspnet/https/${npm_package_name}.pem --ssl-key $HOME/.aspnet/https/${npm_package_name}.key",
 //#else
-    "start": "ng serve --port 5002",
+      "start": "ng serve --port 5002",
```
Comment (TanayParikh) [inline src/content/Angular-CSharp/ClientApp/package.json:12]:
```suggestion
    "start": "ng serve --port 5002",
```

**Review by TanayParikh — APPROVED** [review]

File: eng/Versions.props (hunk 3 above)
Comment (javiercn) [reply eng/Versions.props:12]: Nah, these are only used here for local development. When built as part of the submodule they get the versions from there

**Pull request merged**